<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>739</width>
    <height>414</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>计算</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>371</width>
      <height>321</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>From</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>To</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Number</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>361</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Additional Option</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Error：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eps_lineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>For HTPS</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="mf_checkBox">
         <property name="text">
          <string>Enalbe MultiFilter</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="percent_checkBox">
         <property name="text">
          <string>Show in Percantage</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="face_checkBox">
         <property name="text">
          <string>Near Surface Caculation</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="ave_checkBox">
         <property name="text">
          <string>Show average value</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Main Option</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>最大值：</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="maxLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>最小值：</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="minLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>切分数量：</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="divLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>颗粒总数：</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="total">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>筛选指标：</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>ParticleResidenceTime</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ParticleXPosition</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ParticleYPosition</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ParticleZPosition</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ParticleXVelocity</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ParticleYVelocity</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ParticleZVelocity</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ParticleDiameter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ParticleTemperature</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ParticleDensity </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ParticleMass </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ParticleNumberinParcel</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>目标参数：</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="aim_comboBox">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Number</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TotalMass</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionMultifillter"/>
   <addaction name="actionExport"/>
   <addaction name="actiondraw"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>拓展</string>
    </property>
    <addaction name="actiondraw"/>
    <addaction name="actionExport"/>
    <addaction name="actionMultifillter"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助(&amp;H)</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actiondraw">
   <property name="text">
    <string>绘图(&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>导出(&amp;E)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionMultifillter">
   <property name="text">
    <string>多重筛选(&amp;M)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
